script.bash

```diff
@@ -31,33 +31,40 @@ createDirIfNotExist() {
 
 copyConfigurationIfNotExist() {
     if [ ! -s ${DATA_DIR}/${CONFIG} ]; then
-        echo -e "${RED}config.ini file doesn't exist (or it is empty)${DEFAULT}"
+        echo -e "${RED}${CONFIG} file doesn't exist or it is empty. ${GREEN}Creating...${DEFAULT}"
         cp ${WORKING_DIR}/${CONFIG_EXAMPLE} ${DATA_DIR}/${CONFIG}
         sudo chmod -R +755 ${DATA_DIR}/${CONFIG}
     fi
 }
 
 copyLoggingIfNotExist() {
     if [ ! -s ${DATA_DIR}/${LOGGING} ]; then
-        echo -e "${RED}logging.ini file doesn't exist (or it is empty)${DEFAULT}"
+        echo -e "${RED}${LOGGING} file doesn't exist or it is empty. ${GREEN}Creating...${DEFAULT}"
         cp ${WORKING_DIR}/${LOGGING_EXAMPLE} ${DATA_DIR}/${LOGGING}
         sudo chmod -R +755 ${DATA_DIR}/${LOGGING}
     fi
 }
 
+installPythonRequirements() {
+    echo -e "${GREEN}Installing requirements in ${LIB_DIR}...${DEFAULT}"
+    source ${LIB_DIR}/venv/bin/activate
+    pip install -r requirements.txt
+    deactivate
+}
```

Comment:
Remove this block, it is supposed to installed by default. It just takes time to check this one step.